### PR TITLE
Added styles for news teaser view.

### LINKED ISSUE #532: Fix formatting on News CT Teaser display
- [x] The node title is displaying outside the page width
- [x] The byline is not showing author and date correctly.  Currently shows blank area with border on top and bottom.  Should be formatted as it is on the main article page.

### PR DESCRIPTION
**Pull request by robgasc**

### CODE REVIEW

File: scss/illinois-framework/_node.news.scss [hunk 1]
```diff
@@ -207,3 +207,20 @@ div#block-views-block-news-block-1 {
     }
   }
 }
+
```
Comment (trubach) [inline scss/illinois-framework/_node.news.scss:218]: @robgasc  - A couple things:

1) Can you group these together in SASS? If you see above, there's a few classes that are grouped with curly braces. I think you should be able to have one .spotlight-list-item declaration, and then the sub class declarations grouped within that.

2) Please add a short comment above these to describe what these styles are for

**Comment by robgasc** [comment]
Should close #532. Builds fine on local.

**Review by trubach — CHANGES_REQUESTED** [review]
See my comments on the _node.news.scss file

File: scss/illinois-framework/_node.news.scss (hunk 1 above)
Comment (robgasc) [reply scss/illinois-framework/_node.news.scss:218]: Take a look; Thanks for the catch. Added a group for the news-list-teaser class, and a comment clarifying the styles are for the teaser.

**Comment by robgasc** [comment]
Also needs https://github.com/web-illinois/illinois_framework_core/pull/52 merged to be functional.

**Review by trubach — APPROVED** [review]

**Pull request merged**